inference-engine/thirdparty/clDNN/src/program_helpers.cpp

```diff
@@ -119,6 +119,12 @@ std::pair<bool, bool> program_helpers::are_layouts_identical(layout const& l1, l
     const auto& l2_pad = l2.data_padding;
     if (l1 == l2)
         return {true, true};
+    // If data is actually 1d along f, the layouts are identical
+    if (l1.data_type == l2.data_type && l1.size == l2.size && l1.data_padding == l2.data_padding &&
+        (l1.size.batch[0] == 1 && l2.size.batch[0] == 1) &&
```

Comment:
It's not needed to check batch size twice, because sizes will be the same at this point

Reply:
Removed.